substra/sdk/backends/remote/backend.py

```diff
@@ -109,24 +111,155 @@ def add(self, spec, exist_ok=False, spec_options=None):
                 exist_ok=exist_ok,
                 spec_options=spec_options,
             )
+        elif asset_type == schemas.Type.ComputePlan:
```

Comment:
I think there should be an option to deactivate the autobatching feature just in case :)

Reply:
Added an option ! The most efficient way to batch will always be if the user does it since he does not have to compute the dependency graph.